Use only the C language

libirqf is written completely in C, but by default, CMake assumes that
both C and C++ are used, and therefore verifies that a C++ compiler is
installed. This may not necessarily be the case for a C-only package
in Buildroot.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/src/CMakeLists.txt
===================================================================
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,5 +1,3 @@
-project(iqrf)
-
 set (iqrf_version_major 0)
 set (iqrf_version_minor 1)
 set (iqrf_version_patch 2)
Index: b/CMakeLists.txt
===================================================================
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,3 +1,5 @@
+project(iqrf C)
+
 cmake_minimum_required(VERSION 2.8.1)
 enable_testing()
 
